test_scripts/API/VehicleData/SubscribeVehicleData/001_SubscribeVD_Success.lua

```diff
@@ -0,0 +1,38 @@
+---------------------------------------------------------------------------------------------------
+-- Description: Check that SDL processes SubscribeVehicleData RPC with <vd_param> parameter
+--
+-- Preconditions:
+-- 1) SDL and HMI are started
+-- 2) SubscribeVehicleData RPC and <vd_param> parameter are allowed by policies
+-- 3) App is registered
+--
+-- In case:
+-- 1) App sends valid SubscribeVehicleData(<vd_param>=true) request to SDL
+-- SDL does:
+-- - a) transfer this request to HMI
+-- 2) HMI sends SubscribeVehicleData response with <vd_param> data to SDL
+-- SDL does:
+-- - a) send SubscribeVehicleData response with (success = true, resultCode = "SUCCESS",
+--   <vd_param> = <data received from HMI>) to App
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/API/VehicleData/common')
+
+--[[ Scenario ]]
+common.Title("Preconditions")
+common.Step("Clean environment and update preloaded_pt file", common.preconditions)
+common.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+common.Step("Register App", common.registerApp)
+
+common.Title("Test")
+for param in common.spairs(common.getVDParams(true)) do
+  common.Title("VD parameter: " .. param)
+  common.Step("RPC " .. common.rpc.sub, common.processSubscriptionRPC, { common.rpc.sub, param })
+end
+for param in common.spairs(common.getVDParams(false)) do
```

Comment:
Please extend the description with this case, but i propose to move this case to separate scripts because this is not a success case.